### PR TITLE
Update Maintainer for Otel Go.

### PR DESCRIPTION
**Pull request by rghetia**
- Replace Rahul Patel with Tyler Yahn as a maintainer.

### CODE REVIEW

**Comment by lizthegrey** [comment]
Rahul, would you like to stay as an approver, or are you rotating off entirely?

**Comment by rghetia** [comment]
> Rahul, would you like to stay as an approver, or are you rotating off entirely?

I am rotating off entirely.

File: CODEOWNERS [hunk 1]
```diff
@@ -14,4 +14,4 @@
 
 * @rghetia @tedsuo @jmacd @paivagustavo @krnowak @lizthegrey @MrAlias
```
Comment (lizthegrey) [inline CODEOWNERS:15]: drop from here too if you're leaving approvers.

**Comment by rghetia** [comment]
> Who's going to be a point of contact for the vanity URLs?
I am still the POC until I provide my replacement.

**Pull request merged**